src/input2.c

```diff
@@ -472,7 +472,10 @@ int addnodeID(Network *net, int n, char *id)
 **--------------------------------------------------------------
 */
 {
-    if (findnode(net,id)) return 0;
+    if (findnode(net,id))
+      return 0;
+    if (strlen(id) >= MAXID)
+      return 214; //
```

Comment:
the Slash signs can be removed :)

Reply:
you are right - I put them there thinking I would actually write a comment about the return code...